projects/cmake/build.sh

```diff
@@ -31,6 +32,7 @@ while echo $1 | grep ^- > /dev/null; do
 -help           <true|false>    : update the help database and build the YAML help generator. Defaults to false.
 -boost_root     string          : specify directory containing Boost headers (e.g. `/usr/include`). Defaults to unset.
 -boost_lib      string          : specify directory containing Boost libraries. (e.g. `/usr/lib`). Defaults to unset.
+-boost_include  string          : specify directory containing Boost includes. (e.g. `/usr/include`). Defaults to unset.
 -static_boost	<true|false>    : link using static Boost libraries. Defaults to false.
```

Comment:
I'm inclined to think that we should replace our own `-boost_root` , `-boost_lib`, and `-boost_include` flags by just recommending that people use the cmake flags (i.e. -DBOOST_ROOT=path`) directly.

This would make requests for help on e.g. stackoverflow directly applicable to our use case.  It also removes an (unnecessary?) layer of indirection between the user and cmake, making what is going on more opaque.